registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/auth/dto/VidDto.java

```diff
@@ -0,0 +1,15 @@
+package io.mosip.registration.processor.core.auth.dto;
+
+import lombok.Data;
+
+/**
+ * 
+ * @author Mamta A
+ *
+ */
+@Data
+public class VidDto {
+
+	/** The virtual ID. */
+	private String vid;
```

Comment:
This also we have to rename to individual Id, the idea is this API can return UIN / VID based on a config

Reply:
Changed Class VidDto to individaulDto with individualId property